apps/prairielearn/src/tests/accessibility/index.test.ts

```diff
@@ -38,6 +38,24 @@ async function loadPageJsdom(url: string): Promise<JSDOM> {
  */
 async function checkPage(url: string) {
   const page = await loadPageJsdom(SITE_URL + url);
+
+  // We need to mimic what Bootstrap will do at runtime for tooltips and popovers:
+  // it will copy the `data-bs-title` attribute to the `aria-label` attribute if
+  // the element doesn't already have an `aria-label` attribute and if its text
+  // content is empty.
+  //
+  // Without this, we'll get a bunch of false positives that don't reflect
+  // the actual behavior at runtime.
```

Comment:
The advantage of `title` in the old implementation was that it just worked for these checks (though a `title` on buttons isn't actually generally accepted to be accessible, I think).